nss-ivory-trolls/WorkforceManagement
# Jb fri tic15

# Description

##  Ticket #15 

Given the user is viewing the list of future training programs
When the user performs a gesture of the past programs affordance
Then the user should see a list of all training programs that ended before the current day


## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

1.  `git fetch --all`
1.  `git checkout jb-friTic15`
1.   open project in visual studio
1.  run visual studio server/compiler
1.   in browser click "Training Programs" option in nav bar
1.   you will see upcoming Training Programs
1.  to see Past Training Programs click "View Past Programs" link
1.  you will now see Past Training Programs
1.  to go back to upcoming Training Programs click "View Upcoming Programs" link
1.  you will now see upcoming Trainign Programs


# Checklist:
